instrumentation/instagocb/README.md

```diff
@@ -114,10 +114,140 @@ Sample Usage
+	transactions := cluster.Transactions()
+	_, err = transactions.Run(func(tac *gocb.TransactionAttemptContext) error {
+
+		// Create new TransactionAttemptContext from instagocb
+		tacNew := cluster.WrapTransactionAttemptContext(tac, instagocb.GetParentSpanFromContext(ctx))
+
+		// Unwrapped collection is required to pass it to transaction operations
+		collectionUnwrapped := collection.Unwrap()
+
+		// Inserting a doc:
+		_, err := tacNew.Insert(collectionUnwrapped, "doc-a", map[string]interface{}{})
+		if err != nil {
+			return err
+		}
+
+		// Getting documents:
+		docA, err := tacNew.Get(collectionUnwrapped, "doc-a")
+		// Use err != nil && !errors.Is(err, gocb.ErrDocumentNotFound) if the document may or may not exist
+		if err != nil {
+			return err
+		}
+
+		// Replacing a doc:
+		var content map[string]interface{}
+		err = docA.Content(&content)
+		if err != nil {
+			return err
+		}
+		content["transactions"] = "are awesome"
+		_, err = tacNew.Replace(collectionUnwrapped, docA, content)
+		if err != nil {
+			return err
+		}
+
+		// Removing a doc:
+		docA1, err := tacNew.Get(collectionUnwrapped, "doc-a")
+		if err != nil {
+			return err
+		}
+		err = tacNew.Remove(collectionUnwrapped, docA1)
+		if err != nil {
+			return err
+		}
+
+		// Performing a SELECT N1QL query against a scope:
+		qr, err := tacNew.Query("SELECT * FROM hotel WHERE country = $1", &gocb.TransactionQueryOptions{
+			PositionalParameters: []interface{}{"United Kingdom"},
+
+			// Unwrapped scope is required here
+			Scope: inventoryScope.Unwrap(),
+		})
+		if err != nil {
+			return err
+		}
+
+		type hotel struct {
+			Name string `json:"name"`
+		}
+
+		var hotels []hotel
+		for qr.Next() {
+			var h hotel
+			err = qr.Row(&h)
+			if err != nil {
+				return err
+			}
+
+			hotels = append(hotels, h)
+		}
+
+		// Performing an UPDATE N1QL query on multiple documents, in the `inventory` scope:
+		_, err = tacNew.Query("UPDATE route SET airlineid = $1 WHERE airline = $2", &gocb.TransactionQueryOptions{
+			PositionalParameters: []interface{}{"airline_137", "AF"},
+			// Unwrapped scope is required here
+			Scope: inventoryScope.Unwrap(),
+		})
+		if err != nil {
+			return err
+		}
+
+		// There is no commit call, by not returning an error the transaction will automatically commit
+		return nil
+	}, nil)
+	var ambigErr gocb.TransactionCommitAmbiguousError
+	if errors.As(err, &ambigErr) {
+		log.Println("Transaction possibly committed")
+
+		log.Printf("%+v", ambigErr)
+		return nil
+	}
+	var failedErr gocb.TransactionFailedError
+	if errors.As(err, &failedErr) {
+		log.Println("Transaction did not reach commit point")
+
+		log.Printf("%+v", failedErr)
+		return nil
+	}
+
+	if err != nil {
+		return err
+	}
+ ```
+ [Full example][fullExample]
+
+[gocb-transactions-example]: https://docs.couchbase.com/go-sdk/2.6/howtos/distributed-acid-transactions-from-the-sdk.html#creating-a-transaction
+
 [fullExample]: ../../example/couchbase/main.go
 
-<!---
+<!-- -
```

Comment:
Why do we have spaces here now? Isn't this going to break the automated version check?
Or is the other way around and now you fixed it?

Reply:
nah 😄,! 

It seems like an accidental addition occurred in my vscode when I pressed (⌘ + /). It added an extra space between the comment lines. 

Although this won't impact the automation check, as our AWK script specifically looks for a line with the prefix "target-pkg-url" and nothing else, I'll revert this change since it wasn't intended. Thanks for catching this!

Reply:
Done.